core/src/apps/common/confirm.py

```diff
@@ -67,7 +67,7 @@ async def info_confirm(
     info: ButtonContent = InfoConfirm.DEFAULT_INFO,
     info_style: ButtonStyleType = InfoConfirm.DEFAULT_INFO_STYLE,
 ) -> bool:
-    await button_request(ctx, code=code)
+    await button_request(ctx, code=code)  # type: ignore
```

Comment:
I did not remove it yet, so we can stay away from rebase on master (and therefore need of force-push) until the very end (so that all the commit IDs linked here are still valid).

If we keep this comment unresolved, we will notice it later and can remove them